### ISSUE TITLE
Output blurry and black and white

### ISSUE THREAD

**Issue by gaikwadabhishek** [cchen156/Learning-to-See-in-the-Dark#71]
Hey, I have collected my own dataset of RAW Images(images are in .dng) using camera phone(OnePlus 6). I found out that my camera sensor's black level is 63. Made appropriate changes in code, but while testing the images are blurry and in black and white. Am I missing out on anything specific for different kind of raw images?

**Comment by cchen156** [comment]
Did you check the input images and ground truth? Are they in proper range? e.g. [0,1]

**Comment by gaikwadabhishek** [comment]
Yes. We didn't make any changes in your model and we are using the train_sony model. And here are the images of the scene : https://drive.google.com/open?id=1gzxPcPP6MDOgg9l4mCqxWnAPkwODU7pj

**Comment by compscikaran** [comment]
How did you find out your phone camera sensor's black level ?

**Comment by mouryareddy** [comment]
Hi @gaikwadabhishek ,
Could you please tell me , How you have taken pictures, I got very good results with your picture.
Thank you

**Comment by gaikwadabhishek** [comment]
Hi @mouryareddy and @compscikaran, I found black level through exiftool and official camera sensor documentation of phones. 
We had a made a camera application on Android that uses @cchen156 's model. We also created our own dataset on OnePlus 6 and applied transfer learning to combine both models, to give best results. Thank you @cchen156, amazing work. Loved working on it! :)

**Comment by mouryareddy** [comment]
Can you share your one plus 6 dataset with me and can you tell me which app you have used for generating raw images in one plus 6. I have used VSCO app for generating raw images on iPhone 11 but images are bit shaky after processed by model.
Please it helps us

**Comment by gaikwadabhishek** [comment]
Hi Mourya,
The data set was more than 40 GB, haven't uploaded it anywhere as such. Me and my project group had used a tripod stand, my phone had the facility to click raw images though normal camera app.

**Comment by mouryareddy** [comment]
@gaikwadabhishek Is it possible for you to upload some subset of your images like 5 or so on google drive and share it , just to check the model performance with my iPhone 11 and Samsung set. I work in Camera team , I wanted to integrate code in mobile phones.
Thank you

**Comment by mouryareddy** [comment]
@gaikwadabhishek You mentioned that you have created an android app using this paper, If possible can you share the source code for that?
